### PR TITLE
1-8 done

### PR DESCRIPTION
**Pull request by nabilwardeh**

### CODE REVIEW

File: src/containers/Bookings.js [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info("to do!");
-  };
+  constructor(props) {
+    super(props);
+    this.state = { searched: false };
+    this.bookingData = null;
```
Comment (kkarimi) [inline src/containers/Bookings.js]: Although as you have spotted you can do it this way, per the React pattern this data belongs to the state, i.e.
`
this.state = { 
  searched: false,
  bookingData: null
 };
`

File: src/components/Results.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      filter: {},
+      rowsSelected: new Set()
+    };
+  }
+
+  orderData = (e) => {
+    if (!this.props.bookingResults) return;
+    let elementId = e.target.id;
+
+    if (this.state.filter[elementId]) {
```
Comment (kkarimi) [inline src/components/Results.js]: I don't believe this `if ()` is ever true as you don't seem to update the `filter` in the state anywhere else? can you double check?

File: src/components/Search.js [hunk 3]
```diff
@@ -1,41 +1,115 @@
...
+        className="form-control"
+        placeholder={this.props.placeholder}
+        name={this.props.name}
+        onChange={this.props.handleInput}
+      />
+    );
+  }
+}
+
+class SearchButton extends Component {
```
Comment (kkarimi) [inline src/components/Search.js]: Using multiple components in a file can be a nice way to condense the number of files, but in a bigger project I'd say one file per component is probably best

File: src/containers/Bookings.js [hunk 4]
```diff
@@ -1,19 +1,49 @@
...
+    } else if (filter.customerName) {
+      this.bookingData = FakeBookings.filter(booking => booking.firstName.includes(filter.customerName));
+    } else {
+      this.bookingData = [...FakeBookings];
+    }
+    this.addNumberOfDays();
+    this.setState({searched: true});
+  };
+
+  addNumberOfDays = () => {
```
Comment (kkarimi) [inline src/containers/Bookings.js]: Good job in breaking down the functions, makes it easier to read and write tests for

File: src/containers/Bookings.js [hunk 5]
```diff
@@ -1,19 +1,49 @@
...
+  };
+
   render() {
     return (
       <div className="App-content">
         <div className="container">
-          <Search search={this.search} />
-          {/* <Results results={this.state.results} /> */}
+          <Search search={this.search} runSearch={this.handleSearchButtonClicks} />
+          <Results bookingResults={this.bookingData} />
```
Comment (kkarimi) [inline src/containers/Bookings.js]: Should be per the commented out line, i.e. 
`<Results results={this.state.results} /> `

**Comment by nabilwardeh** [comment]
Finished 9-10

**Pull request closed without merging**